### DISCUSSION TITLE
GLIBC library version error

### DISCUSSION THREAD

**Discussion by vukvucic** [directus/directus#6222] [Archive]
We have been trying to install the new Directus 9 Node.Js version on our shared hosting to have a play with it but we seem to be getting an error in regards to the version of the glibc library.

After chatting with the hosting providers they told us that they only support glibc-2.17 because they are running on Cent-OS 7 and it seems the Directus 9 npm package has a dependency on glibc-2.25.

Is there any way to configure this or has anyone else found a different solution? 🙏

<img width="1653" alt="Screen Shot 2021-06-12 at 1 20 03 pm" src="https://user-images.githubusercontent.com/25982108/121768560-1efd7680-cba2-11eb-8d1e-4f829002ffc3.png">

**Comment by rijkvanzanten** [comment]
I believe you can npm install using `--build-from-source` which will trigger npm to rebuild the native binaries (including argon2) to use your current system 🤔

**Reply by vukvucic** [reply]
Thank yeah that's what I thought but seems to still tell me I have the wrong version.

I will continue taking a look thanks heaps!

**Reply by chrisjung-dev** [reply]
You may find your answer here: https://lab.uberspace.de/guide_directus/#installation
